gatsby-plugin-theme-ui/theme-loader.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path')
+const { getOptions } = require('loader-utils')
+
+module.exports = function () {
+  const opts = getOptions(this)
+  const filename = path.join(opts.dirname, 'themes.json')
+  const themes = require(filename)
+
+  return `
+    export const themes = [
+      ${themes.map(filename => `require('${filename}').default`)}
+    ]
+
+    export default {
+      themes,
+    }
+
+    if (module.hot) module.hot.accept()
```

Comment:
Hot reloading doesn't seem to pick up changes. This was an attempt to make it work

Reply:
you'd need `this.addDependency` to re-trigger the loader iirc https://webpack.js.org/api/loaders#thisadddependency